scripts/runlogwatch.sh

```diff
@@ -5,9 +5,12 @@ LOG_DIR="/shared/log/ironic/deploy"
 
 mkdir -p "${LOG_DIR}"
 
+# shellcheck disable=SC2034
 python3 -m pyinotify -e IN_CLOSE_WRITE -v "${LOG_DIR}" |
-    while read -r path _action file; do
-        echo "************ Contents of ${path}/${file} ramdisk log file bundle **************"
-        tar -xOzvvf "${path}/${file}" | sed -e "s/^/${file}: /"
-        rm -f "${path}/${file}"
+    while read -r event dir mask maskname filename filepath pathname wd; do
```

Comment:
Can we add in a comment here what the pyinotify event looks like? It seems that we drastically go from 3 things to 7.

Please also add the same in the PR description, and also maybe why it ever worked with the 3 args before?